.nycrc.yaml

```diff
@@ -0,0 +1,3 @@
+# # This option avoid an wrap by nyc that affects our expected async events.
+# # See: https://github.com/clinicjs/node-clinic-bubbleprof/pull/382
+use-spawn-wrap: true
```

Comment:
it's needed because `tap@15` uses a `nyc` version that wraps the tests using external libraries and affects the `async_hooks` events. 

Further info: https://github.com/istanbuljs/nyc/pull/1169/files